lib/node_modules/@stdlib/math/base/special/cabsf/benchmark/julia/benchmark.jl

```diff
@@ -2,7 +2,7 @@
 #
 # @license Apache-2.0
 #
-# Copyright (c) 2021 The Stdlib Authors.
+# Copyright (c) 2024 The Stdlib Authors.
```

Comment:
It isn't necessary to change these copyright years. If the file already exists, the copyright year can remain as is. Otherwise, this just creates diff noise.

Reply:
i wasn't sure of this here, noted.